conf/defaults.yaml

```diff
@@ -231,16 +228,13 @@ topology.multilang.serializer: "org.apache.storm.multilang.JsonSerializer"
 topology.shellbolt.max.pending: 100
 topology.skip.missing.kryo.registrations: false
 topology.max.task.parallelism: null
-topology.max.spout.pending: null
+topology.max.spout.pending: null  # TODO: We dont need this any more
```

Comment:
Yes back-pressure is addressed. Described in the design doc. A couple little things left realaed.. 1) Ensure metrics get updated when retrying to publish to downstream Q that is full. 2) Remove max.spout.pending